kubernetes/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+# This Kubernetes configmap file is intended for local environment deployment only.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spark-manager-config
+  namespace: cdm-jupyterhub
+data:
+  KUBE_NAMESPACE: "cdm-jupyterhub"
+  SPARK_IMAGE: "spark-standalone:latest" 
+  SPARK_IMAGE_PULL_POLICY: "IfNotPresent"
+  POSTGRES_URL: "postgres:5432"
+  POSTGRES_DB: "hive"
+  POSTGRES_USER: "hive"
```

Comment:
I don't see anything in the k8s deployment about deploying a postgres / hive db

Reply:
I removed them - I copied the deployment file from jupyterhub. I missed removing them.